templates/partials/list-content.html

```diff
@@ -0,0 +1,21 @@
+{% from "components/question/_macro.njk" import onsQuestion %}
+
+{% set title = content.content.title %}
+{% set contents = content.content.contents %}
+
+{% call onsQuestion({
+  "id": question.id,
+  "title": title,
+}) %}
+  {%- if content.list and content.list.list_items -%}
+    {% set list = content.list %}
+    <div class="ons-u-mb-m">
+      {% include 'partials/summary/list-summary.html' %}
+    </div>
+  {%- endif -%}
+
+  {% for description in contents %}
+    <p>{{ description["description"] }}</p>
+  {% endfor %}
```

Comment:
All contents should work, this is only doing paragraphs. We should just be calling out to `partials/contents.html`.

Reply:
Using partial now.

Reply:
We should add more properties that contents support. It wasn't using it yet no tests failed. Even though it has now changes, no tests verify it.